build.gradle

```diff
@@ -10,9 +10,14 @@ buildscript {
 }
 apply plugin: 'net.minecraftforge.gradle'
 //Only edit below this line, the above code adds and enables the necessary things for Forge to be setup.
+apply plugin: 'eclipse'
```

Comment:
Is this a leftover from your local setup?

Reply:
Yeah i guess you dont need these